src/components/Table/components/Row.tsx

```diff
@@ -93,8 +92,13 @@ function RowImpl<R extends Kinded, S>(props: RowProps<R>): ReactElement {
   const rowStyleCellCss = maybeApplyFunction(row as any, rowStyle?.cellCss);
   const levelIndent = style.levels && style.levels[level]?.rowIndent;
 
-  const rowCss = {
+  const containerCss = {
     ...Css.add("transition", "padding 0.5s ease-in-out").$,
```

Comment:
**nit/question:** This timing feels a bit slow. Have you been able to work with Design to define the proper behaviors/timings for the drag and drop?

Reply:
Nope, sure haven't, but I made it a bit snappier